sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensor.java

```diff
@@ -165,7 +165,9 @@ private void saveMeasures(SensorContext context,
       String filePath = entry.getKey();
       InputFile cxxFile = fs.inputFile(fs.predicates().hasPath(filePath));
       if (cxxFile != null) {
-        Collection<Measure> measures = entry.getValue().createMeasures();
+        CxxUtils.LOG.debug("Saving for file '{}'", filePath);
```

Comment:
it is in connection with the change below, an exception was thrown and i did not had any file information, therefore i added this line to debug log